posix.mak

```diff
@@ -43,6 +44,9 @@ REBASE = MYBRANCH=`git rev-parse --abbrev-ref HEAD` &&\
 CHANGE_SUFFIX = \
  for f in `find "$3" -iname '*.$1'`; do\
   mv $$f `dirname $$f`/`basename $$f .$1`.$2; done
+ 
+ # Set to 1 in the command line to minify css files
+ CSS_MINIFY=
```

Comment:
Just for documentation. Recall that in makefiles setting a variable in the command line overrides manual setting with `=`.